newrelic-agent/src/main/resources/newrelic.yml

```diff
@@ -293,17 +286,18 @@ common: &default_settings
 
   # Cross Application Tracing adds request and response headers to
   # external calls using supported HTTP libraries to provide better
-  # performance data when calling applications monitored by other New Relic Agents.
+  # performance data when calling applications monitored by other New Relic agents.
   #
   # Distributed tracing is replacing cross application tracing as the default
   # means of tracing between services. To continue using cross application
   # tracing, enable it with `cross_application_tracer.enabled = true` and
   # `distributed_tracing.enabled = false`
   cross_application_tracer:
 
-    # Set to true to enable cross application tracing.
-    # Default is false.
-    enabled: false
+    # Set to false to disable cross-application tracing.
+    # Default is true.
+    enabled: true
```

Comment:
This setting should be false. Distributed tracing replaced cross application tracing as the default.

Reply:
@tbradellis OK, I will change cross application tracing to "false," but this concerns me that this has been true while at the same time, distributed tracing was also true.

Reply:
It's my understanding that cross application tracing has been false for a long time. I just pulled down the latest Java agent and the config file is showing "false" so I'm not sure how this got changed to "true."

Reply:
@tbradellis I resolved the conflict for cross application tracing by changing it to the value of "false."